### ISSUE TITLE
Units and conversion factors

### ISSUE THREAD

**Issue by BobHanson** [MolSSI/QC_JSON_Schema#30]
I have implemented in Jmol a general Java class for handling QC-JSON units. It is working smoothly in my first tries at QC-JSON writer and reader. 

See [org.qcshema](https://sourceforge.net/p/jmol/code/HEAD/tree/trunk/Jmol/src/org/qcschema/)

I suggest that this be the start of a Java library. 

Who else is interested in working on this with me?

**Comment by berquist** [comment]
I just encountered some momentary confusion regarding units comparing some old and new Psi4 output due to https://github.com/psi4/psi4/pull/1800/files#diff-3abbb4f37a59cc2e6cfbff3f4cb73b312dfd94044a19f38b00bf7ae140db13e7L90, and saw that QCSchema doesn't have units yet. I know that units and conversion factors are a part of QCElemental, but the schema needs to at least say what fields have what units.

**Comment by adam-grofe** [comment]
Agreed. Is there any plan to add what units should be used in the documentation at least?
